docs/src/pages/demos/chips/ChipsPlayground.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { Fragment } from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Reply:
Then... I don't now because I did not found a way to test, because the option to change for TS is not being displayed. Plis can you help me to resolve this problem, maybe exists a conflict with` import MarkdownElement from 'docs/src/modules/components/MarkdownElement`